### PR TITLE
fix(V1.2.x.Recurring.Transactions): update using master patches

### PR DESCRIPTION
**Pull request by stef145g**
Fix recurring transactions and other relevant fixes

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: f21f6a69afce6ea5c876960827634380a437b84d
  * Your subject line is longer than 72 characters

Guidelines are available at https://github.com/moneymanagerex/moneymanagerex/wiki/GitCop

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: f21f6a69afce6ea5c876960827634380a437b84d
  * Your subject line is longer than 72 characters

Guidelines are available at https://github.com/moneymanagerex/moneymanagerex/wiki/GitCop

---
This message was auto-generated by https://gitcop.com

File: src/billsdepositspanel.cpp [hunk 1]
```diff
@@ -451,7 +449,10 @@ wxString mmBillsDepositsPanel::getItem(long item, long column)
     case COL_STATUS:
         return bill.STATUS;
     case COL_CATEGORY:
-        return bill.CATEGNAME;
+        if (bill.has_split())
+            return _("Split.......");
```
Comment (vomikan) [inline src/billsdepositspanel.cpp]: I've suggest the same changes as here:
https://github.com/moneymanagerex/moneymanagerex/pull/476

~~~~~~~
-    CATEGNAME = Model_Category::full_name(r.CATEGID, r.SUBCATEGID);
+    if (!m_bill_splits.empty())
+    {
+        for (const auto& entry : m_bill_splits)
+            CATEGNAME += (CATEGNAME.empty() ? " * " : ", ")
+            + Model_Category::full_name(entry.CATEGID, entry.SUBCATEGID);
+    }
+    else
+        CATEGNAME = Model_Category::full_name(r.CATEGID, r.SUBCATEGID);
~~~~~~

File: src/mmpanelbase.h [hunk 2]
```diff
@@ -46,6 +46,8 @@ class mmListCtrl : public wxListCtrl
 
     virtual wxListItemAttr* OnGetItemAttr(long row) const;
     wxString BuildPage(const wxString &title) const;
+    int GetColumnWidthSetting(const int& column_number, int default_size = wxLIST_AUTOSIZE);
```
Comment (vomikan) [inline src/mmpanelbase.h]: We should avoid references for simple types.

**Comment by stef145g** [comment]
I will re-do the patches as these were directly cherry picked from the master branch.
Then re-submit the new changes

**Pull request closed without merging**